src/main/python/setup.py

```diff
@@ -49,19 +48,7 @@
     name=ARTIFACT_NAME,
     version=ARTIFACT_VERSION_SHORT,
     description='SystemDS is a distributed and declarative machine learning platform.',
-    long_description='''SystemDS is a versatile system for the end-to-end data science lifecycle from data integration,
-     cleaning, and feature engineering, over efficient, local and distributed ML model training,
-     to deployment and serving.
-     To facilitate this, bindings from different languages and different system abstractions provide help for:
-     (1) the different tasks of the data-science lifecycle, and 
-     (2) users with different expertise. 
-
-
-     These high-level scripts are compiled into hybrid execution plans of local, in-memory CPU and GPU operations, 
-     as well as distributed operations on Apache Spark. In contrast to existing systems 
-     - that either provide homogeneous tensors or 2D Datasets - and in order to serve the entire
-     data science lifecycle, the underlying data model are DataTensors, i.e.,
-     tensors (multi-dimensional arrays) whose first dimension may have a heterogeneous and nested schema.''',
+    long_description=open('README.md').read(),
```

Comment:
:+1:

Reply:
Once the license is added in the README, you might want to skip that amount of lines here to shorten the description.

Reply:
Do you think it is worth it? I like the simplicity.

Reply:
then keep it!